core/src/main/java/dk/alexandra/fresco/framework/builder/NumericBuilder.java

```diff
@@ -31,6 +33,14 @@
 
   Computation<SInt> known(BigInteger value);
 
+  default List<Computation<SInt>> known(List<BigInteger> value) {
```

Comment:
To the best of my knowledge known is just a lazy way of sharing constant input data. I still have not seen a place where known really is used for something essential. I recently pushed a branch to eliminate every call to known apart from LP/DEA and CompareToZero/sign (lacking a concrete implementation of this).

So instead of adding more usages to I would recommend removing it altogether - and at least not add more utlity functions for working with known

Reply:
Known is supposed to be used by application developers when they have a publicly known value which is used within the application. Most of our interfaces (if not all) does not support using just a BigInteger, but instead a Computation<SInt>. This means that if known is not present, one would have to input the value which requires communication. This is not preferred. 
I stick with keeping it. It also makes it easier to write tests instead of first having to input.

Reply:
First of all. The production API should not support easy testing that would be wrong.

There might be cases (especially in the boolean arithmetic) where the constants can be used in the computations. I am not avid supporter of removing the known per se. But the list of BigInteger is just lazy testers. If the is the desired way of writing Tests then by all means have it in the tests. But do not include it in the production code.